model-optimizer/mo/front/extractor.py

```diff
@@ -805,10 +805,14 @@ def add_input_op_input_port_without_data(graph: Graph, node_id: str, input_op, e
 
 
 def add_input_op_input_port_with_data(graph: Graph, node_id: str, input_op, edge_attrs: dict):
-    input_data_node = input_op.create_node_with_data()
-    input_node = input_data_node.in_node()
-    graph.add_edge(input_data_node.id, node_id, **edge_attrs)
-    update_ie_fields(graph.node[input_node.id])
+    input_node = input_op.create_node(edge_attrs=edge_attrs)
```

Comment:
Check that graph is not on front stage, so data nodes will be really created

Reply:
Both updated functions called for graph after shape inference, but I can such assert.

Reply:
Done.